client-spark/spark3/src/main/java/org/apache/spark/shuffle/RssShuffleManager.java

```diff
@@ -214,16 +221,6 @@ public RssShuffleManager(SparkConf conf, boolean isDriver) {
                     .unregisterRequestTimeSec(unregisterRequestTimeoutSec)
                     .rssConf(rssConf));
     registerCoordinator();
-    // fetch client conf and apply them if necessary and disable ESS
-    if (isDriver && dynamicConfEnabled) {
-      Map<String, String> clusterClientConf =
-          shuffleWriteClient.fetchClientConf(
```

Comment:
Can you delete the `fetchClientConf` method in `shuffleWriteClient` ?

Reply:
This method is still used by MR and TEZ clients.

Reply:
Got it.